src/main/java/org/example/stellog/review/domain/Review.java

```diff
@@ -0,0 +1,37 @@
+package org.example.stellog.review.domain;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.example.stellog.room.domain.Room;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Review {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "review_id")
    private Long id;
```

이와같이 DB 컬럼명을 명확하게 지정해주는게 좋을 것 같아요.
이 부분은 이전 리뷰에서도 말한부분입니다!

Reply:
넵 이 부분 수정했고 다음부턴 까먹지 않겠습니다! 알려주셔서 감사합니다~